app/activity/[id]/layout.tsx

```diff
@@ -0,0 +1,17 @@
+'use client'
```

Comment:
Do we need to use client component, even though we don't have client only code ...?

Reply:
Actually, I don't need layout.tsx!  The header and footer will be rendered from the root directly. I removed this file!